src/PKSim.Presentation/DTO/Mappers/ExpressionProfileToExpressionProfileDTOMapper.cs

```diff
@@ -38,14 +39,19 @@ public ExpressionProfileDTO MapFrom(ExpressionProfile expressionProfile)
             Icon = _moleculePropertiesMapper.MoleculeIconFor(expressionProfile.Molecule),
             Species = expressionProfile.Species,
             Category = expressionProfile.Category,
-            MoleculeName = expressionProfile.MoleculeName,
+            MoleculeName = moleculeNameFor(expressionProfile),
             AllMolecules = _usedMoleculeRepository.All(),
             AllSpecies = _speciesRepository.All(),
             MoleculeType = _moleculePropertiesMapper.MoleculeDisplayFor(expressionProfile.Molecule),
          };
 
-         dto.AddExistingExpressionProfileNames(_projectRetriever.Current.All(PKSimBuildingBlockType.ExpressionProfile).AllNames().Except(new[] {expressionProfile.Name}));
+         dto.AddExistingExpressionProfileNames(_projectRetriever.Current.All<ExpressionProfile>().AllNames().Except(new[] {expressionProfile.Name}));
          return dto;
       }
+
+      private string moleculeNameFor(ExpressionProfile expressionProfile)
+      {
+         return string.Equals(expressionProfile.MoleculeName, CoreConstants.DEFAULT_EXPRESSION_PROFILE_MOLECULE_NAME) ? string.Empty : expressionProfile.MoleculeName;
```

Comment:
This is the only change  that was required not to have a moleculeName property. I can take that